src/WebJobs.Extensions.DurableTask/DurabilityProvider.cs

```diff
@@ -408,5 +408,15 @@ public virtual bool ValidateDelayTime(TimeSpan timespan, out string errorMessage
             errorMessage = null;
             return true;
         }
+
+        /// <summary>
+        ///  Returns true if the stored connection string, ConnectionName, matches the input DurabilityProvider ConnectionName.
+        /// </summary>
+        /// <param name="durabilityProvider">The DurabilityProvider used to check for matching connection string names.</param>
+        /// <returns>A boolean indicating whether the connection names match.</returns>
+        public virtual bool ConnectionNameMatches(DurabilityProvider durabilityProvider)
```

Comment:
Any particular reason for making this public instead of internal? I'd prefer to make it internal, if possible, since it doesn't look like any outside code needs direct access to this method.